README.md

```diff
@@ -1,2 +1,9 @@
 # angular-clearable
-Directive for adding a clear affordance to input fields
+
+Angular directive for adding a clear affordance to input fields
+
+## Use
+
+`<input type="text" data-clearable>`
+
+This will create an icon on the right of the input field that when selected clears the input field. The icon used depends on fontawesome.
```

Comment:
Is it really worth including fontawesome for this one icon?  It's just an X, you could draw one with two CSS rectangles pretty trivially.